vignettes/continuous-integration.Rmd

```diff
@@ -86,6 +86,6 @@ usethis::use_github_action("lint-project")
 
 ### Super-Linter
 
-`lintr` powers R lints for [Super-Linter](https://github.com/github/super-linter) and [MegaLinter](https://oxsecurity.github.io/megalinter/latest/),
+`lintr` powers R lints for [Super-Linter](https://github.com/github/super-linter) and [MegaLinter](https://megalinter.io/latest/),
```

Comment:
Can you split this off to fix the link rot issue on main?

Reply:
Good idea. Done in https://github.com/r-lib/lintr/pull/1779